sysboundary/setbyuser.cpp

```diff
@@ -288,7 +288,7 @@ namespace SBC {
          std::array<bool, 6> isThisCellOnAFace;
          determineFace(isThisCellOnAFace, mpiGrid, id);
 
-         int max = 6;
+         uint max = 6;
```

Comment:
wouldn't this be better done with just a loop from 0 to <6, and a break?

Reply:
True. Good sugestion. Done.